lib/dev/panel.js

```diff
@@ -227,11 +227,13 @@ createView.define(Panel, (panel, document) => {
   const frame = document.createElement("iframe");
   setAttributes(frame, {
     "sandbox": "allow-scripts",
-    // It would be great if we could allow remote iframes for sandboxing
-    // panel documents in a content process, but for now platform implementation
-    // is buggy on linux so this is disabled.
+    // We end up using chrome iframe with forced message manager
+    // as fixing a swapFrameLoader seemd like a giant task (see
+    // Bug 1075490).
     // "remote": true,
-    "type": "content",
+    // "type": "content",
+    "type": "chrome",
```

Comment:
> Do we have a test that remote content is not used?

I don't think that's relevant to this change, but yes we options are validated:

https://github.com/mozilla/addon-sdk/blob/master/lib/dev/panel.js#L158-L164